src/com/executors/executeservice/GetName.java

```diff
@@ -0,0 +1,28 @@
+package com.executors.executeservice;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.util.concurrent.Callable;
+
+public class GetName implements Callable<String> {
+    private static final String LOGIN_DB = "login";
+    private static final String PASSWORD_DB = "password";
+    private static final String NAME_DB = "nameBD";
+    private Connection connect;
+    private int id;
+
+    public GetName(int id) throws Exception {
+        connect = DriverManager.getConnection("jdbc:mysql://localhost/" + NAME_DB + "?characterEncoding=utf8", LOGIN_DB, PASSWORD_DB);
```

Comment:
"jdbc:mysql://localhost/" + NAME_DB + "?characterEncoding=utf8" this all better replace by constant.